velox/functions/prestosql/BinaryFunctions.h

```diff
@@ -436,4 +436,107 @@ struct ToIEEE754Bits32 {
   }
 };
 
+template <bool lpad, bool isAscii, typename TOut, typename TIn>
```

Comment:
It doesn't seem like isAscii template parameter is being used in the function logic. Why is it relevant ?